webclient/src/websocket/commands/SessionCommands.ts

```diff
@@ -25,6 +25,9 @@ export class SessionCommands {
       case WebSocketConnectReason.PASSWORD_RESET:
         SessionCommands.updateStatus(StatusEnum.CONNECTING, 'Connecting...');
         break;
+      case WebSocketConnectReason.TEST_CONNECTION:
+        webClient.testConnect({ ...options });
```

Comment:
Would passing options be the same as what you're doing?

Reply:
Yes, it's just a convention to ensure immutability.